.werft/jobs/build/prepare.ts

```diff
@@ -51,16 +49,14 @@ async function createVM(werft: Werft, config: JobConfig) {
         "GOOGLE_APPLICATION_CREDENTIALS": GCLOUD_SERVICE_ACCOUNT_PATH,
         "TF_VAR_cert_issuer": config.certIssuer,
         "TF_VAR_preview_name": config.previewEnvironment.destname,
-        "TF_VAR_vm_cpu": `${cpu}`,
-        "TF_VAR_vm_memory": `${memory}Gi`,
+        "TF_VAR_with_large_vm": `${config.withLargeVM}`,
```

Comment:
Awesome - please update [our docs](https://www.notion.so/gitpod/Preview-Environments-6debd359591b43688b52f76329d04010#7508fbb4e85c46ee93b18a6b0c09fb8c) once this is merged so we don't mention TF_VAR_vm_cpu and TF_VAR_vm_memory anymore.